test/filters/range_test.rb

```diff
@@ -22,12 +22,27 @@ def test_filter_invalid_if_has_no_options
 
   def test_to_hash_should_return_query
     filter = Elasticquery::Filters::Range.new :year, lte: 1, gte: 0
-    assert_equal({query: {filtered: {filter: {and: {range: {year: {lte: 1, gte: 0}}}}}}}, filter.to_hash)
+    assert_equal({query: {filtered: {filter: {and: [{range: {year: {lte: 1, gte: 0}}}]}}}}, filter.to_hash)
```

Comment:
Line is too long. [107/80]<br>Space inside { missing.<br>Space inside } missing.